test/sample_number_v4.json

```diff
@@ -1 +1 @@
-{"numberWithPercentage":{"v":"Default","t":1,"i":"642bbb26","p":[{"o":0,"v":"80%","p":80,"i":"ad5f05a7"},{"o":1,"v":"20%","p":20,"i":"786b696f"}],"r":[{"o":0,"a":"Custom1","t":10,"c":"sajt","v":"=sajt","i":"216987c8"},{"o":1,"a":"Custom1","t":12,"c":"2.1","v":"<2.1","i":"a900bc23"},{"o":2,"a":"Custom1","t":13,"c":"2,1","v":"<=2,1","i":"2c85f73d"},{"o":3,"a":"Custom1","t":10,"c":"3.5","v":"=3.5","i":"ae86baf5"},{"o":4,"a":"Custom1","t":14,"c":"5","v":">5","i":"c6924001"},{"o":5,"a":"Custom1","t":15,"c":"5","v":">=5","i":"8090543a"},{"o":6,"a":"Custom1","t":11,"c":"4.2","v":"<>4.2","i":"2691fade"}]},"number":{"v":"Default","t":1,"i":"5ced27a9","p":[],"r":[{"o":0,"a":"Custom1","t":11,"c":"5","v":"<>5","i":"a41938c5"}]}}
```

Comment:
I guess file name should be v5 here and all other test files as well.

Reply:
Fixed